lib/apiservers/portlayer/swagger.yml

```diff
@@ -859,3 +920,46 @@ definitions:
       state:
         type: string
         enum: ["RUNNING", "STOPPED"]
+  VolumeRequest:
+    type: object
+    required:
+      - Name
+      - Store
+      - Driver
+    properties:
+      Name:
+        type: string
+      Driver:
```

Comment:
On second thought, I will avoid this for now. the refactor would not be too bad. But in the event that we want it eventually it can still be there. It does not adversely effect anything in the design.